apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md

```diff
@@ -0,0 +1,68 @@
+---
+eleventyNavigation:
+    key: 'Numeric Stepper - Form'
+    parent: numeric-stepper
+---
+
+## Overview
+The purpose of numeric steppers as form fields is to offer users flexibility in inputting numeric values, maintaining a consistent form field appearance in form interfaces. This allows users to easily adjust and enter numerical data with ease.
```

Comment:
```suggestion
The purpose of numeric steppers as form fields is to offer users flexibility in inputting numeric values, while maintaining a consistent form field appearance in form interfaces. This allows users to easily enter and adjust numerical data.
```

Reply:
this small change got me looking at this 😄 https://www.grammarly.com/blog/while-vs-whilst/#:~:text=Whilst%20and%20while%20are%20two,That's%20the%20main%20difference.